submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/files/FileConfig.kt

```diff
@@ -12,5 +13,7 @@ class FileConfig {
     fun fileMapper() = FilesMapper()
 
     @Bean
-    fun fileServiceFactory(ftpClient: FtpClient) = FileServiceFactory(ftpClient)
+    fun fileServiceFactory(
+        @Qualifier("userFilesFtpClient") ftpClient: FtpClient,
```

Comment:
I would create this string "userFilesFtpClient" as a constant and put it when beans are declared so we do not accdentaly change the bean name